### ISSUE TITLE
Feature request: Store login informations of the websites

### ISSUE THREAD

**Issue by merlinuwe** [nguyenvanduocit/obsidian-open-gate#14]
Thanks for your great plugin.

I use some webpages with login credentials. Is there a way to store them?

**Comment by nguyenvanduocit** [comment]
can you provide me some website for example.

**Comment by merlinuwe** [comment]
Thanks for your reply. Yes, these are the main sites I need:

![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/4138869/89f79f14-c471-47ab-a8ce-33db7f734d3d)


![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/4138869/457b1ce1-aed7-4d62-a974-89a7ad399118)

**Comment by merlinuwe** [comment]
I just installed your new release and put this in a new note:

```gate  
https://herakles.webuntis.com/WebUntis/index.do#/basic/login
height:900
profile:account-1
```

It renders only to a small window:

![image](https://github.com/nguyenvanduocit/obsidian-open-gate/assets/4138869/502c773c-78d5-4ab9-b2a8-0c988b1c4813)

(On the right pane, it renders the full website.)

Can I use profile: to give the site my user credentials? How do I have to give the credentials? I tried 

profile:account-1

and

profile:username@passwort

but this does not work.
